experiments/benchmarks/land.jl

```diff
@@ -622,8 +622,7 @@ function setup_and_solve_problem(; greet = false)
 
     # Define timestepper and ODE algorithm
     stepper = CTS.ARS343()
-    norm_condition = CTS.MaximumError(FT(1e-8))
-    conv_checker = CTS.ConvergenceChecker(; norm_condition = norm_condition)
+    conv_checker = nothing
```

Comment:
the default is nothing, so we could just not pass `conv_checker` at all :)

Reply:
good call!